leptonai/api/photon.py

```diff
@@ -130,6 +135,11 @@ def run_remote(
         photon_id=id,
         resource_requirement=types.ResourceRequirement.make_resource_requirement(
             resource_shape=resource_shape,
+            replica_cpu=replica_cpu,
```

Comment:
Do old versions of workspaces accept these resource requirements?

Reply:
Seems no. We need support for shape update.

Reply:
Discussed with @xiang90 offline, these are supported in all workspaces.